### DISCUSSION TITLE
Issue with fileUpload progressBarTemplate

### DISCUSSION THREAD

**Discussion by chrishj59** [primefaces/community#797] [PrimeReact] [answered]
Hi,
I am having an issue with FileReader using uploadHandler and  progressBarTemplate.
It is probably due to incorrect usage on my part.

I get a TS error '...' expected.ts(1005)
The call to 

 ```xml
<FileUpload
							ref={fileUploadRef}
							name="xtraderCat"
							uploadHandler={onUploadHandler}
						
							progressBarTemplate{<ProgressBar value={progress} />}
							auto
							accept="text/xml"
							chooseLabel="Select Category file"
							maxFileSize={1000000}
						/>
```

If I try to call a simple function in progressBarTemplate

```js
const progressTemplate = () => {
		return (<ProgressBar value={progress} />)
	}
```
I get the same TS error.

Please let me know how I should be calling progressBarTemplate.
Thank you

**Comment by melloware** [comment, marked as answer]
It is a bug I just opened a ticket: https://github.com/primefaces/primereact/issues/5664

Once its fixed you will do this...

```
progressBarTemplate={(opts) => <ProgressBar value={opts.progress} />}
```
